teams/rust-analyzer.toml

```diff
@@ -1,16 +1,14 @@
-name = "wg-rls-2"
+name = "wg-rust-analyzer"
```

Comment:
We should probably rename this as well, no?

Reply:
Oh I missed that